ofctrl/fgraphSwitch.go

```diff
@@ -96,31 +97,39 @@ func (self *OFSwitch) DefaultTable() *Table {
 	return self.tableDb[0]
 }
 
-// Create a new group. return an error if it already exists
-func (self *OFSwitch) NewGroup(groupId uint32, groupType GroupType) (*Group, error) {
-	// check if the group already exists
-	if self.groupDb[groupId] != nil {
-		return nil, errors.New("group already exists")
+// NewGroup creates a new group; when using cache, returns an error if the group ID already exists, otherwise returns the
+// group object; when not using cache, returns the new group object， and the caller should ensure the groupID is not
```

Comment:
```suggestion
// group object; when useCache is false, a new group object is returned and the caller should ensure the groupID is not
```

Reply:
Thanks.

Not related to this PR: you could select multiple lines when reviewing code on time.